### PR TITLE
Localization - Afrikaans [afr]

### PR DESCRIPTION
**Pull request by JonathanAspeling**
## :scroll: Description!
I added English to Afrikaans Translation. Which includes:

- Store Listing in Afrikaans
- Prompts in Afrikaans
- UI text to Afrikaans

_Elected to not translate quotes - they tend to not read/mean the same as when in original English_

## :green_heart: How did you test it?
Manual - Tested with Afrikaans spell checkers and colleagues to ensure grammar is correct.

## :camera_flash: Screenshots / GIFs

[image](https://user-images.githubusercontent.com/42679894/119255104-22609c00-bbba-11eb-89a2-4ccccf3aee9d.png)

### CODE REVIEW

**Comment by JonathanAspeling** [comment]
Ah - thank goodness for automated testing - I think I see the issue.

**Comment by JonathanAspeling** [comment]
Hehe cool not sure where I'm missing things :D But welcome to let me know and I'll fix and commit

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -23,4 +23,5 @@
     <string name="language_sk" translatable="false">Slovák</string>
     <string name="language_es" translatable="false">Español</string>
     <string name="language_tr" translatable="false">Türkçe</string>
+    <string name="language_af" translatable="false">Afrikaans</string>
```
Comment (alisonthemonster) [inline app/src/main/res/values/strings.xml:26]: Oh you made me realize we have these files in the `app` module in addition to the `strings` module, I'll fix that in another PR!

File: strings/src/main/res/values-af/prompts.xml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    <string-array name="prompts">
+            <item>Wat het jou laat glimlag vandag?</item>
+            <item>Wat is daar voor om dankbaar te wees in die huidige kamer?</item>
+            <item>Wat maak jou dankbaar om lewendig te wees?</item>
+            <item>Wie het jou onlangs gehelp?</item>
+            <item>Wie het jou onder hulle vlerk gevat?</item>
+            <item>Wie het jou iets nuuts geleer?</item>
+            <item>Wie het jy lief?</item>
+            <item>Wie het jou lief?</item>
+            <item>Het jy dalk 'n vriend vir wie jy dankbaar is?</item>
```
Comment (alisonthemonster) [inline strings/src/main/res/values-af/prompts.xml]: These are the issue you are having with CI! You need to escape the apostrophe characters like so:
```
<item>Het jy dalk \'n vriend vir wie jy dankbaar is?</item>
```

File: strings/src/main/res/values-af/strings.xml [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+    <string name="change_time">Verander u daaglikse herinneringstyd</string>
+    <string name="time_title">Pas Herinnering Kennisgewings Aan</string>
+    <string name="theme_change">Kies uit meer as 20 verskillende temas en kleurskemas</string>
+    <string name="notifs_off">Kennisgewings vir daaglikse herinneringe is af</string>
+    <string name="notifs_on">Kennisgewings vir daaglikse herinneringe is an</string>
+    <string name="notifs">Kennisgewings vir daaglikse herinneringe</string>
+    <string name="notifs_name">Kennisgewings</string>
+    <string name="appearance">Voorkoms</string>
+    <string name="about">App Inligting</string>
+    <string name="select_a_theme">Kies 'n tema</string>
```
Comment (alisonthemonster) [inline strings/src/main/res/values-af/strings.xml]: Here too!

File: strings/src/main/res/values-af/strings.xml [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+    <string name="theme">Tema</string>
+    <string name="open_source">Oopbronlisensies</string>
+    <string name="open_source_info">Lees oopbronbiblioteeklisensies</string>
+    <string name="permission_export">Toestemming is nodig om data uit te voer</string>
+    <string name="error_parsing">Kon nie lêer ontleed nie</string>
+    <string name="file_not_csv">Lêer moet 'n CSV wees</string>
```
Comment (alisonthemonster) [inline strings/src/main/res/values-af/strings.xml]: Here 👋

File: strings/src/main/res/values-af/strings.xml [hunk 5]
```diff
@@ -0,0 +1,111 @@
...
+    <string name="error_creating_csv_file">Kon nie CSV-lêer op toestel skep nie</string>
+    <string name="no_app_found">Geen program gevind om hierdie lêer oop te maak nie</string>
+    <string name="copied">Gekopieer</string>
+    <string name="security">Privaatheid</string>
+    <string name="lock_title">Verkry toegang tot Presently</string>
+    <string name="lock_summary">Presently is gesluit, ontsluit om voort te gaan</string>
+    <string name="lock_setting_summary_on">Presently sluit met die biometriese skandeerder van u toestel.</string>
+    <string name="lock_setting_summary_off">Presently kan sluit met die biometriese skandeerder van u toestel (as dit deur die toestel ondersteun word).</string>
+    <string name="fingerprint_error_lockout_too_many">Te veel pogings tot biometriese verifikasie. Probeer asseblief oor 'n paar minute weer.</string>
```
Comment (alisonthemonster) [inline strings/src/main/res/values-af/strings.xml]: Here!

File: strings/src/main/res/values-af/strings.xml [hunk 6]
```diff
@@ -0,0 +1,111 @@
...
+    <string name="continue_to_exit">Gaan voort</string>
+    <string name="are_you_sure">Is jy seker jy wil teruggaan?</string>
+    <string name="unsaved_text">U inskrywing is nie gestoor nie en u veranderinge sal verlore gaan!</string>
+    <string name="version_number">Weergawe Nommer</string>
+    <string name="daily">Daagliks</string>
+    <string name="weekly">Weekliks</string>
+    <string name="every">Op elke verandering</string>
+    <string name="dropbox_summary_off">Meld aan met u Dropbox-rekening om u inskrywings outomaties te rugsteun</string>
+    <string name="dropbox_summary_on">U inskrywings word in Dropbox gerugsteun</string>
+    <string name="dropbox_setting_title">Maak 'n rugsteun van u data met Dropbox</string>
```
Comment (alisonthemonster) [inline strings/src/main/res/values-af/strings.xml]: This one too!

File: strings/src/main/res/values-af/strings.xml [hunk 7]
```diff
@@ -0,0 +1,111 @@
...
+    <string name="dropbox_backup_frequency">Gereeldheid van Dropbox Rugsteun</string>
+    <string name="one_time_export_summary">Voer u inskrywings as CSV uit na u toestel</string>
+    <string name="one_time_export">Voer inskrywings uit na CSV</string>
+    <string name="import_summary">Kies 'n gerugsteunde CSV-lêer om in te voer</string>
```
Comment (alisonthemonster) [inline strings/src/main/res/values-af/strings.xml]: And finally this one!

File: strings/src/main/res/values-af/strings.xml [hunk 8]
```diff
@@ -0,0 +1,111 @@
...
+    <string name="day_of_week_title">Datumstyl</string>
+    <string name="max_lines_title">Tydlyn Inskrywing Lengte</string>
+    <string name="max_lines_summary">Aantal lyne in die tydlyn. Die volledige teks is sigbaar as u op die inskrywing klik.</string>
+    <string name="saturday">Saterdag</string>
+    <string name="sunday">Sondag</string>
+    <string name="monday">Maandag</string>
+    <string name="day_of_week_summary">Stel die eerste dag van die week in die kalenderaansig</string>
+    <string name="first_day_of_week_title">Eerste dag van die week</string>
+    <string name="share_milestone">Ek het vir %1$d dae dankbaarheids inskrywings gemaak! Sluit by my aan deur die Presently App. #PresentlyApp</string>
+    <string name="share_app_text">Oefen dankbaarheid met hierdie eenvoudige, altyd gratis en private dankbaarheidsjoernaal app</string>
```
Comment (alisonthemonster) [inline strings/src/main/res/values-af/strings.xml:110]: I also noticed there are a few translations missing here, would you mind adding these?
```
    <string name="app_language_title">Application language</string>
    <string name="app_language_summary">Set default app language</string>
    <string name="loading_lang">Loading language, please wait</string>
    <string name="show_quote_off">Gratitude quotes will not be shown on entry page</string>
    <string name="show_quote_on">Gratitude quotes will be shown on entry page</string>
    <string name="inspiration_title">Inspirational quotes</string>
```

**Review by alisonthemonster — CHANGES_REQUESTED** [review]
Left some comments about some string character escpaing and a few missing translations. Thank you so much for doing this translation and for figuring out my new module structure! I'll be updating the guide for translators hopefully soon!

**Comment by JonathanAspeling** [comment]
Awesome it's a pleasure! Will attend to it first chance!

File: app/src/main/res/values/strings.xml (hunk 1 above)
Comment (JonathanAspeling) [reply app/src/main/res/values/strings.xml:26]: 100% Not sure if sorted as yet - think I have the latest version of code base :D Let me know if I have an action point here - not seeing something specific

File: strings/src/main/res/values-af/prompts.xml (hunk 2 above)
Comment (JonathanAspeling) [reply strings/src/main/res/values-af/prompts.xml]: Brilliant! I think this is to match with Regex? - Have made the change will push up soonest

File: strings/src/main/res/values-af/strings.xml (hunk 3 above)
Comment (JonathanAspeling) [reply strings/src/main/res/values-af/strings.xml]: Cheers! Got em!

File: strings/src/main/res/values-af/strings.xml (hunk 4 above)
Comment (JonathanAspeling) [reply strings/src/main/res/values-af/strings.xml]: Got them! Thanks!

File: strings/src/main/res/values-af/strings.xml (hunk 5 above)
Comment (JonathanAspeling) [reply strings/src/main/res/values-af/strings.xml]: Cheers thanks for having taken the time to highlight! Appreciate

File: strings/src/main/res/values-af/strings.xml (hunk 6 above)
Comment (JonathanAspeling) [reply strings/src/main/res/values-af/strings.xml]: Perfect got it!

File: strings/src/main/res/values-af/strings.xml (hunk 7 above)
Comment (JonathanAspeling) [reply strings/src/main/res/values-af/strings.xml]: Shot! Got it!

File: strings/src/main/res/values-af/strings.xml (hunk 8 above)
Comment (JonathanAspeling) [reply strings/src/main/res/values-af/strings.xml:110]: With pleasure! Done

**Comment by JonathanAspeling** [comment]
Yass! CI Passed :D hope you're happy with the changes @alisonthemonster Again blown away by what you've put in place and developed 👏 

Apologies for the long the long turn around time here 😄  Baby and a recent promotion to Data Engineer has kept me so busy! Been sweating blood and tears balancing things and inevitably dropped some balls :D

Shout if there's anything else - feel free to @ me

File: app/src/main/res/values/strings.xml (hunk 1 above)
Comment (alisonthemonster) [reply app/src/main/res/values/strings.xml:26]: You're okay, I never actually did remove the duplicated strings 🤦‍♀️

**Comment by alisonthemonster** [comment]
Thank you so much for the changes and congrats on baby and promo! 👏 I've kicked off a CI build, will approve when its green!

**Review by alisonthemonster — APPROVED** [review]

**Pull request merged**